### ISSUE TITLE
Transfert to leviia impossible if past target folder has been moved

### ISSUE THREAD

**Issue by CKoes** [nextcloud/android#11612]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

1)Push to cloud a file from mobile
2) delete folder in cloud
3) push new file to cloud> past target folder is indicated
4) when trying to change hierarchy> error

### Expected behaviour

If past target folder doesn't existe anymore show root folder of cloud

### Cause of error
```java
Exception in thread "Thread-7" java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.owncloud.android.datamodel.OCFile.getRemotePath()' on a null object reference
    at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:229)
    at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:399)
    at java.lang.Thread.run(Thread.java:923)

```

### App information
* ID: `com.leviia.client`
* Version: `130230290`
* Build flavor: `generic`

### Device information
* Brand: `samsung`
* Device: `a20e`
* Model: `SM-A202F`
* Id: `RP1A.200720.012`
* Product: `a20eeea`

### Firmware
* SDK: `30`
* Release: `11`
* Incremental: `A202FXXU4CVK1`

### Android version

11

### Device brand and model

Samsung

### Stock or custom OS?

Stock

### Nextcloud android app version

11

### Nextcloud server version

11

### Using a reverse proxy?

I don't know

**Comment by joshtrichards** [comment]
Duplicate of #11207

**Comment by joshtrichards** [comment]
Fixed by #11410
